crypto/crypto_aes.c

```diff
@@ -57,7 +59,7 @@ useaesni(void)
 	static int aesnigood = -1;
 	uint8_t key[32];
 	uint8_t ptext[16];
-	size_t i;
+	uint8_t i;
```

Comment:
Do the warnings go away if you keep this as a size_t, but add a `& 0xff` where it's used?  That would make for saner code.